src/acceptance/app/dynamic_policy_test.go

```diff
@@ -0,0 +1,127 @@
+package app
+
+import (
+	"acceptance/config"
+	"acceptance/helpers"
+	"fmt"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/cf"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/generator"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var _ = Describe("AutoScaler dynamic policy", func() {
+	var appName string
+	var appGUID string
+	var instanceName string
+	var initialInstanceCount int
+
+	BeforeEach(func() {
+		instanceName = generator.PrefixedRandomName("autoscaler", "service")
+		createService := cf.Cf("create-service", cfg.ServiceName, cfg.ServicePlan, instanceName).Wait(cfg.DefaultTimeoutDuration())
+		Expect(createService).To(Exit(0), "failed creating service")
+	})
+
+	JustBeforeEach(func() {
+		appName = generator.PrefixedRandomName("autoscaler", "nodeapp")
+		countStr := strconv.Itoa(initialInstanceCount)
+		createApp := cf.Cf("push", appName, "--no-start", "-i", countStr, "-b", cfg.NodejsBuildpackName, "-m", cfg.NodeMemoryLimit, "-p", config.NODE_APP, "-d", cfg.AppsDomain).Wait(cfg.DefaultTimeoutDuration())
+		Expect(createApp).To(Exit(0), "failed creating app")
+
+		guid := cf.Cf("app", appName, "--guid").Wait(cfg.DefaultTimeout)
+		Expect(guid).To(Exit(0))
+		appGUID = strings.TrimSpace(string(guid.Out.Contents()))
+
+	})
+
+	AfterEach(func() {
+		unbindService := cf.Cf("unbind-service", appName, instanceName).Wait(cfg.DefaultTimeoutDuration())
```

Comment:
These aren't matched properly.
If my BE runs, createService is executed but no bind. This will do an unbind but leave my service behind.